docs/plugins.md

```diff
@@ -248,10 +248,10 @@ vs
 
 This also works with scoped packages
 
-```js
+```json
 {
-  presets: ["@org/babel-preset-name"], // actual package
-  presets: ["@org/name"] // shorthand name
+  "presets": ["@org/babel-preset-name"], // actual package
```

Comment:
Oh right so true. We might need to remove all comments from configs then.. I wish we could just strip them out before the clipboard haha